dask_planner/src/sql/logical.rs

```diff
@@ -48,56 +48,40 @@ impl PyLogicalPlan {
     }
 }
 
+/// Convert a LogicalPlan to a Python equivalent type
+fn to_py_plan<T: From<LogicalPlan>>(current_node: Option<&LogicalPlan>) -> PyResult<T> {
```

Comment:
This function accepts any generic type `T` that also implements `From<LogicalPlan>`